torch/csrc/api/include/torch/nn/options/activation.h

```diff
@@ -19,6 +19,16 @@ struct ELUOptions {
 
 // ============================================================================
 
+/// Options for SELU functional and module.
+struct SELUOptions {
+  SELUOptions() {}
+
+  /// can optionally do the operation in-place. Default: False
+  TORCH_ARG(bool, inplace) = false;
```

Comment:
Please make it constructor argument with default value, as @yf225 asked [here](https://github.com/pytorch/pytorch/pull/27035#discussion_r329657107)